Prace_domowe/Praca_domowa4/Grupa2/Adrian_Kaminski/PD4.ipynb

```diff
@@ -0,0 +1,1928 @@
+{
```

Comment:
<p>W przypadku gdy mamy do czynienia z obrazkami, można dokonywać standaryzacji w sposób "naturalny", tzn. jeśli minimalną wartością natężenia pixela jest 0 a maksymalną 16, to po prostu dzielimy cały zbiór danych przez 16, tak żeby wartości były z przedziału [0,1]</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/237/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>